content/en/platform/corda/4.12/community/getting-set-up.md

```diff
@@ -19,26 +19,24 @@ title: Getting set up for CorDapp development
 # Getting set up for CorDapp development
 
 There are four pieces of required software for CorDapp development:
-* Java 8 JDK
+* Java 17 JDK
 * IntelliJ IDEA
 * Git
 * Gradle
 
 {{< note >}} Visit the [platform support matrix]({{< relref "release-platform-support-matrix.md" >}}) to check the latest third-party application versions supported by Corda. {{< /note >}}
 
-## Installing the Java 8 JDK
+## Installing the Java 17 JDK
 
-Install the Java 8 JDK. Corda requires at least version 8u171, but do not currently support Java 9 or higher for this version of Corda.
+Install the Java 17 JDK. 
 
-Corda has been tested against the following Java builds:
+Corda 4.12 has been tested against the following Java builds:
 
-  * [Oracle JDK](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
-  * [Amazon Corretto](https://aws.amazon.com/corretto/)
-  * [Red Hat’s OpenJDK](https://developers.redhat.com/products/openjdk/overview/)
-  * [Zulu’s OpenJDK](https://www.azul.com/)
+  * [Oracle JDK 17.0.9](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html)
```

Comment:
again with consistency, I think mention the version above and then just talk about generic JDK builds that are compatible, the user can then find that for themselves:

> [Oracle JDK]